src/components/OffCanvas/OffCanvas.tsx

```diff
@@ -12,6 +12,13 @@ import OffCanvasContent from './OffCanvasContent';
 import * as baseTheme from './OffCanvas.scss';
 
 export type Mode = 'slide' | 'push' | 'reveal';
+export type WidthType = 'Small' | 'Medium' | 'Large' | string;
+
+export enum Width {
```

Comment:
@HiirenP the 'Small' | 'Medium' | 'Large' width options should apply a class, not a value. The display of the drawer needs to be defined and controlled from the scss file.